src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const firstDiscount = 20;
+  const secondDiscount = 50;
```

Comment:
Think about more meaningful variable names, what is first discount, second discount?